crates/ty_completion_eval/completion-evaluation-tasks.csv

```diff
@@ -20,6 +20,6 @@ scope-existing-over-new-import,main.py,0,474
 scope-prioritize-closer,main.py,0,2
 scope-simple-long-identifier,main.py,0,1
 tstring-completions,main.py,0,1
-ty-extensions-lower-stdlib,main.py,0,7
+ty-extensions-lower-stdlib,main.py,0,8
```

Comment:
this change is because there is now another `ty_extensions` function (`reveal_mro`) that currently has a higher rank than `typing.reveal_type` in autocomplete suggestions, but would have a lower rank in an ideal world: https://github.com/astral-sh/ruff/blob/64ab79e5721ec6fdd2182fbf9d39a26534ccca43/crates/ty_completion_eval/truth/ty-extensions-lower-stdlib/main.py#L2